CHANGELOG.md

```diff
@@ -5,29 +5,28 @@ CHANGELOG
 -----
 
 **ENHANCEMENTS**
-- Add possibility to specify multiple instance types for the same compute resource.
+- Add possibility to specify multiple EC2 instance types for the same compute resource.
 - Add support for adding and removing shared storages at cluster update by updating `SharedStorage` configuration.
-- Add new configuration parameter `DeletionPolicy` for EFS and FSx for Lustre shared storage
-  to support storage retention on deletion.
-- Add support for Slurm Accounting.
-- Add support for on-demand capacity reservations.
-- Add support for specifying the supported IMDS version in cluster and build image configurations via the `Imds/ImdsSettings` property.
+- Add new configuration parameter `DeletionPolicy` for EFS and FSx for Lustre shared storage to support storage retention on deletion.
+- Add new configuration to enable accounting functionality in Slurm.
```

Comment:
We should mention the configuration name then: `Scheduling/slurmSettings/Database`.